invoice/views.py

```diff
@@ -89,15 +89,7 @@ class InvoiceCreateView(SuccessMessageMixin, CreateView):
     success_message = 'Invoice was created successfully.'
 
     def get_success_url(self):
-        return reverse_lazy('invoice-update', kwargs={'pk': self.object.id})
-
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
-        if self.request.POST:
-            context['invoice_item_form'] = InvoiceItemForm(self.request.POST)
-        else:
-            context['invoice_item_form'] = InvoiceItemForm()
-        return context
+        return reverse('invoice-update', kwargs={'pk': self.object.id})
```

Comment:
@iTitus why can't we do `reverse_lazy` here?

Reply:
_lazy is a pointless indirection. At this time all views are setup and reverse lookup works